File: 2 - Operators/Solution.kt
import java.lang.Math.round

fun main(args: Array<String>) {
        val mealCost = readLine()!!.toDouble()
        val tipPercent = readLine()!!.toInt()
        val taxPercent = readLine()!!.toInt()

        val tip = tip_percent * meal_cost / 100
        val tax = tax_percent * meal_cost / 100

        println((tip + tax + meal_cost).roundToInt())
}

File: 2 - Operators/Solution.cpp
#include <cmath>
#include <iostream>

using namespace std;

int main() {
    string tmp;

    getline(cin, tmp);
    double mealCost = stod(tmp);

    getline(cin, tmp);
    int tipPercent = stoi(tmp);

    getline(cin, tmp);
    int taxPercent = stoi(tmp);

    double tip = tipPercent * mealCost / 100;
    double tax = taxPercent * mealCost / 100;

    int totalCost = (int) round(tip + tax + mealCost);
    cout<<totalCost;

    return 0;
}
